src/client/src/ui/common/regions/actions.ts

```diff
@@ -3,9 +3,9 @@ import { RegionSettings } from '../../../types'
 
 export interface Region {
   id: string
-  isTearedOff: boolean
-  container: React.ComponentClass
-  settings: RegionSettings
+  isTearedOff?: boolean
+  container?: React.ComponentClass
```

Comment:
We should remove this? Unless this is a WIP step.

Reply:
WIP step